spec/features/items/show_spec.rb

```diff
@@ -5,11 +5,15 @@
     before :each do
       @merchant_1 = create(:merchant)
       @item_1 = create(:item, user: @merchant_1, inventory: 15)
-    end
 
-    it "I can see all the information for the item" do
+      @order_item_1 = create(:fulfilled_order_item, item: @item_1, created_at: 3.days.ago, updated_at: 1.days.ago)
+      @order_item_2 = create(:fulfilled_order_item, item: @item_1, created_at: 2.days.ago, updated_at: 1.days.ago)
+      @order_item_3 = create(:fulfilled_order_item, item: @item_1, created_at: 4.days.ago, updated_at: 1.days.ago)
+
       visit item_path(@item_1)
+    end
 
+    it "I can see all the information for the item" do
```

Comment:
I would add a test for a visitor and a regular user to check that they will not see a link for "Add to Cart".

Reply:
Adds test to see who can see the Add to Cart button.
Thank you, Stella!